Sources/TuistGenerator/Generator/Generator.swift

```diff
@@ -132,7 +132,7 @@ public class Generator: Generating {
 
     public func generateProject(at path: AbsolutePath) throws -> (AbsolutePath, Graph) {
         let config = try graphLoader.loadConfig(path: path)
-        try environmentLinter.lint(config: config).printAndThrowIfNeeded()
+        try environmentLinter.lint(config: config, at: path).printAndThrowIfNeeded()
```

Comment:
@mollyIV have you considered adding a `path: AbsolutePath` attribute to the `Config` model where we set the path where the config has been loaded from. That way we don't have to modify the signature of the methods to pass down the path.

Reply:
👋 Pedro, 

actually, I did. I though that `Config` model should contain only the properties that a user can define 🤔 If that's not a case, I will do the suggested change:

```
// Config.swift

public init(compatibleXcodeVersions: CompatibleXcodeVersions = .all,
                cloud: Cloud? = nil,
                generationOptions: [GenerationOptions],
                path: Path) { ... }
```

Thanks!

Reply:
👋 In fact, that's the case for the `Config.swift` model in the `ProjectDescription` framework. That's the user-facing one. There's another `Config.swift` file inside the `TuistCore` framework that is an internal representation. That one can contain extra attributes, like the path to the directory where the config has been loaded from. We do exactly the same for `Project.swift` and `Workspace.swift`.

Reply:
I implemented the suggested changes 🙇